test/internet/test-http-dns-fail.js

```diff
@@ -4,7 +4,6 @@
  * should trigger the error event after each attempt.
  */
 
-var common = require('../common');
```

Comment:
There's this too but I think that's it: 

https://github.com/nodejs/node/blob/68dc69a382692f424c53b75724e684a406b195a1/test/common.js#L142-L151